### ISSUE TITLE
Model play area regions for accessibility

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#150]
@terracoda has provided a diagram that will specify descriptions for the balloon by location.  When this is modeled, it will also help easily look up descriptions for the balloon based on its position.

![bs-grid-new 1](https://cloud.githubusercontent.com/assets/6396244/15691713/98afee72-2747-11e6-9ef4-77ca21192324.png)

**Comment by jessegreenberg** [comment]
Regions added to the model.  I think the image above is not at the same aspect ration of the dev bounds, but the balloon is still able to reach each region.

Here is the grid visualized in the simulation:
![grid](https://cloud.githubusercontent.com/assets/6396244/15694367/458e694a-275a-11e6-9a03-ecaa4a74bad1.PNG)

It can be seen in future versions with the ?dev query parameter.

**Comment by jessegreenberg** [comment]
To clarify, balloon descriptions will be based off of the center of the location of the center of the balloon, and the center of the balloon can touch each region.

**Comment by terracoda** [comment]
@jessegreenberg, thanks for posting here and creating the ?dev parameter. This is very useful! I can actually see how big the balloon steps are in real time. This will help a lot with descriptions! 

More clarification:
- Dark vertical lines:  center of the balloon at the "Jump to locations"
- Black dots on edges: center of balloon at extreme edges (will need sound effects here)
- Green vertical area: region of play area exposed when Wall is removed

Sorry, the aspect ration was working off an older file to try and save time.
Also, the "Close to Sweater" jump spot looks like it might be too close to sweater. I'll fix that.

**Comment by jessegreenberg** [comment]
> Also, the "Close to Sweater" jump spot looks like it might be too close to sweater

Sorry, how is this?
![near](https://cloud.githubusercontent.com/assets/6396244/15730391/90b40b08-2829-11e6-8055-23ec0d7603f9.PNG)

> Black dots on edges: center of balloon at extreme edges (will need sound effects here)

Ah, very cool! Didn't realize that.

**Comment by terracoda** [comment]
Hi @jessegreenberg , I updated the diagram in the google sheet. It should be to scale now, sorry for the confusion. The mark above for the JS location is the same as in the diagram now. I made other slight adjustments to the regions as well. I will try to post the diagram here.

**Comment by terracoda** [comment]
Here we go:
![bs-grid-new-2-scale](https://cloud.githubusercontent.com/assets/1902116/15809080/4a6ad176-2b63-11e6-90c3-4cdba7171be5.png)

**Comment by terracoda** [comment]
Legend for description grid diagram:

- Darker Blue regions: X & Y regions
- Top labels: X release descriptions
- Left labels: Y release descriptions
- Black dots: mark center of balloon at a boundary
- Bottom labels: sample drag description moving left
- Grey words: filler drag descriptions
- Green bar: marks rest of Play Area when Wall is removed
- Vertical grey lines: Jump to locations (JS, JC, JN, JW)
- Yellow oval: balloon marker, centered dot marks location of balloon on the grid

**Comment by jessegreenberg** [comment]
Thanks @terracoda, I will make updates to the model as necessary.

**Comment by terracoda** [comment]
Thanks @jessegreenberg and "dev parameter" that displays the model is really handy. It is way easier to see how many intermittent descriptions we need. Things like "closer", "almost there". I think I can reduce.

**Comment by jessegreenberg** [comment]
I am glad the in-sim visualization is helpful @terracoda!  Since this has been implemented, closing.  If the play area model needs to be tweaked, we can create new issues.
